src/main/webapp/WEB-INF/themes/mobile/resource/css/interface.css

```diff
@@ -70,12 +50,12 @@
   zoom: 1;
 }
 
-.bold {
-  font-weight: bold;
+.600 {
```

Comment:
I guess it's legit in HTML5, but it sure looks odd.

Reply:
Actually, that's unintentional! Search & replace side-effect.  I'll fix this.